SCons/Taskmaster/Job.py

```diff
@@ -239,505 +228,523 @@ def start(self):
         self.taskmaster.cleanup()
 
 
-# Trap import failure so that everything in the Job module but the
-# Parallel class (and its dependent classes) will work if the interpreter
-# doesn't support threads.
-try:
-    import queue
-    import threading
-except ImportError:
-    pass
-else:
-    class Worker(threading.Thread):
```

Comment:
@bdbaddog - Let me know if you want a PR that is *just* the removal of this check and then another that is just the de-indent. Then I could try to rebase this PR above those after they merge, and maybe the real diffs will be a lot easier to see.

Reply:
This might help avoid the need for that. Here is an "ignore whitespace" view of the very first commit in the stack, where I did the removal and de-indent: https://github.com/acmorrow/scons/commit/8c8052c95d4645a584b4332e11aabe0843446678?diff=split&w=1. That way you can see what really changed there in the various schedulers is, effectively, nothing.

Then, review the PR without that bottom commit, like: https://github.com/SCons/scons/pull/4464/files/8c8052c95d4645a584b4332e11aabe0843446678..e4e128a70ea7659b95c5edcc713436683d9a2cca

Reply:
This file is fine as is.